Rakefile

```diff
@@ -11,7 +12,9 @@ Rake::TestTask.new(:test) do |test|
   test.warning = false
 end
 
-task :default => "wwtd:local"
+task :default => ["rubocop", "wwtd:local"]
+
+RuboCop::RakeTask.new
```

Comment:
prefer `sh "rubocop --disaply-cap-names"`which makes it able to see what cop needs to be modified